### PR TITLE
add babelmixr2

### PR DESCRIPTION
**Pull request by mattfidler**
This adds the commercial options from `babelmixr2`

### CODE REVIEW

File: MixedModels.md [hunk 1]
```diff
@@ -232,6 +238,8 @@ These topics are closely related because there are few available analytical meth
 - Other software: [Zelig Project](https://zeligproject.org/).
 - Other software: [MixWild/MixRegLS](https://voices.uchicago.edu/hedeker/mixwild_mixregls/) for scale-location modeling.
 - Other software: [MixedModels.jl](https://github.com/JuliaStats/MixedModels.jl) for mixed models in Julia.
+- Other software: [Monolix](https://monolix.lixoft.com/) for ODE based mixed models (commerical).
+- Other software: [NONMEM](https://www.iconplc.com/innovation/nonmem/)  for ODE based mixed models (commerical).
```
Comment (palday) [inline MixedModels.md:242]:
```suggestion
- Other software: [NONMEM](https://www.iconplc.com/innovation/nonmem/) for ODE based mixed models (commercial).
```

**Review by palday — APPROVED** [review]

**Review by bbolker — CHANGES_REQUESTED** [review]
My markup is clunky here, sorry. [] are insertions, [DELETE ...] and [CHANGE ...] should speak for themselves. These are suggestions (except for the fixed typo) rather than mandatory requirements.

allows `r pkg("nlmixr2")` models to [be] translated and run in either the commercial tool 
  [Monolix](https://monolix.lixoft.com/) or [NONMEM](https://www.iconplc.com/innovation/nonmem/) and then read[s] the results 
  back in to create a standardized `nlmixr2` fit object. This fit object runs the diagnostics in `nlmixr2` and [DELETE the] compares [them]
  to the ones output in the commercial software to "validate" the fit object [CHANGE represents the → against the] output of the commercial tool.  
  It also interfaces with free tools [CHANGE as well like → such as] `r pkg("PKNCA)` [CHANGE for automatically using observed Pharmcokinetic → in order to use observed pharmacokinetic] (PK) data 
  [CHANGE for → to derive]  initial estimates of PK models.

**Comment by bbolker** [comment]
PS thanks for the contribution!

**Comment by mattfidler** [comment]
> PS thanks for the contribution! (I forget: is there a PK task view either in existence or in prep??)

There is a PK task view; https://cran.r-project.org/web/views/Pharmacokinetics.html

**Comment by mattfidler** [comment]
I hope I got everything you suggested.  Thanks for the edit!

**Comment by bbolker** [comment]
didn't fix typo in "Pharmcokinetic" (which I also wouldn't capitalize).  (Sorry, I would fix this myself, but not 100% clear on how to make edits on PRs ...)

**Comment by mattfidler** [comment]
Great catch

**Pull request merged**